Makefile

```diff
@@ -0,0 +1,7 @@
+all: modulo_lib.o
+
+modulo_lib.o: modulo_lib.cc modulo_lib.h 
+	g++ -c modulo_lib.cc -o $@
```

Comment:
You can use the make automatic variable $< to reference the first prerequisite (modulo_lib.cc in this case).

Reply:
Wow, I didn't know that. Thanks! Done.